index.css

```diff
@@ -69,5 +69,6 @@ body {
 }
 
 .winner { /* ADJUST BORDER COLOR FOR THE TEAM WITH MORE POINTS*/
```

Comment:
We might want to update the description here--and remove the border if we're not going to use it.